docs/docsite/rst/dev_guide/testing/sanity/no-list-cmp.rst

```diff
@@ -0,0 +1,8 @@
+Sanity Tests » no-list-cmp
+==========================
+
+The ``cmp`` function has been removed in python 3.
```

Comment:
All instances of the word "Python" that aren't in a code block should be capitalized.